Gemfile

```diff
@@ -1,3 +1,3 @@
-source "http://rubygems.org"
+source "https://stitchfix01.jfrog.io/stitchfix01/api/gems/eng-gems/"
```

Comment:
This is a public repo and gem.

https://rubygems.org/gems/fittings

Reply:
🤦

Reply:
Restored in e071985da27dd7715a20f86a8945312ae641fa86.